src/Type/Definition/FieldDefinition.php

```diff
@@ -56,6 +59,15 @@ class FieldDefinition
     /** @var array<int, Argument> */
     public array $args;
 
+    /**
+     * Callback for resolving field value given parent value.
```

Comment:
Inaccurate

Reply:
Yeah, sorry. I just copied resolver fn logic and forgot to change descriptions. :) Fixed.